### DISCUSSION TITLE
How to install 3.9.16 update

### DISCUSSION THREAD

**Discussion by Savmash** [mhdzumair/MediaFusion#265] [General]
I'm new to stremio,rd,mediafusion. There is a zip file, how to configure it?
Also, the addon status is offline at times (netify) and it doesn't work at this time
Thanks,

**Comment by mhdzumair** [comment]
The zip file is for Kodi. For stremio goto https://mediafusion.elfhosted.com/configure and setup as you preferred then click on install or copy the installation URL and paste it in the stremio.
